sdk/core/azure-core-tracing-opencensus/src/samples/AsyncListKeyVaultSecrets.md

```diff
@@ -1,23 +1,24 @@
-# List Key Vault Secrets with Azure Core Tracing OpenCensus asynchronously
+# Asynchronously list Key Vault Secrets with Azure Core Tracing OpenCensus 
  
-Following documentation describes instructions to run a sample program for listing secrets asynchronously in a Key Vault with tracing instrumentation for Java SDK libraries.
+Following documentation describes instructions to run a sample program for asynchronously creating and listing secrets of a Key Vault with tracing instrumentation.
 
 ## Getting Started
+Sample uses **[opencensus-impl][opencensus_impl]** as implementation package and **[Zipkin Exporter][zipkin_exporter]** as exporter.
 
-### Adding the Azure client library for Key Vault secrets package to your project:
+### Adding dependencies to your project:
 [//]: # ({x-version-update-start;com.azure:azure-security-keyvault-secrets;current})
 ```xml
 <!-- Add KeyVault Secrets dependency without Netty HTTP client -->
 <dependency>
-  <groupId>com.azure</groupId>
-  <artifactId>azure-security-keyvault-secrets</artifactId>
-  <version>4.0.0</version>
-  <exclusions>
-    <exclusion>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-core-http-netty</artifactId>
-    </exclusion>
-  </exclusions>
+    <groupId>com.azure</groupId>
```

Comment:
Should we list out adding KeyVault Secrets without the Netty HttpClient or just have a message to include Secrets, possibly linking to the Secrets README for further reading.

Reply:
Have been thinking about this, will do this update 👍

Reply:
updated.